Orange/widgets/visualize/owscoringsheetviewer.py

```diff
@@ -510,15 +515,32 @@ def _extract_data_from_model(self, classifier):
         all_risks = 1 / (1 + np.exp(-all_scaled_scores))
 
         self.attributes = attributes
+        self._base_coefficients = coefficients
+        self._base_all_scores = all_scores.tolist()
+        self._base_all_risks = (all_risks * 100).tolist()
         self.coefficients = coefficients
         self.all_scores = all_scores.tolist()
         self.all_risks = (all_risks * 100).tolist()
         self.domain = classifier.domain
 
         # For some reason when leading the model the scores and probabilities are
         # set for the wrong target class. This is a workaround to fix that.
+        self._sort_attributes_by_importance()
         self._adjust_for_target_class()
 
+    def _sort_attributes_by_importance(self):
+        """
+        Sorts the attributes in descending order of importance (absolute coefficient value)
+        by combining the attributes and coefficients into tuples and sorting them.
+        """
+        combined = list(
```

Comment:
No need to call `list` here. You can pass a `zip` object to `sorted`; `sorted` will work on any iterable.